pkg/fanal/analyzer/pkg/apk/apk_test.go

```diff
@@ -300,7 +402,7 @@ func TestParseApkInfo(t *testing.T) {
 			t.Errorf("%s : can't open file %s", testname, v.path)
 		}
 		scanner := bufio.NewScanner(read)
-		gotPkgs, gotFiles := a.parseApkInfo(scanner)
+		gotPkgs, gotFiles := a.parseApkInfo(scanner, &analyzer.AnalysisOptions{KeepSystemInstalledFiles: true})
```

Comment:
`KeepSystemInstalledFiles` shouldn't always be true. We should pass the value for each test case. Otherwise, we cannot test the case where `KeepSystemInstalledFiles` is false.